examples/sort.py

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/env python
+
+# Copyright 2022 NVIDIA Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import argparse
+
+import numpy
+from benchmark import run_benchmark
+from legate.timing import time
+
+import cunumeric
```

Comment:
A nitpick: in other tests, we import `numpy` as `np` and `cunumeric` as `num`. I guess you can follow the same convention (which will help to reduce clutters in this test from a long package name).

Reply:
I will replace that